.github/workflows/ci.yml

```
@@ -0,0 +1,140 @@
+name: CI
+
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+  pull_request:
+
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        numfocus_nightly: [false]
+        os: ["ubuntu-latest"]
+        pyarrow: ["0.13.0", "0.14.1", "0.15.0", "0.16.0", "0.17.1", "nightly"]
+        python: ["3.6", "3.7", "3.8"]
+        include:
+          - numfocus_nightly: true
+            os: "ubuntu-latest"
+            pyarrow: "0.17.1"
+            python: "3.8"
+          - numfocus_nightly: false
+            os: "macos-latest"
+            pyarrow: "0.17.1"
+            python: "3.8"
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      # CI setup
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Conda Bootstrap
+        uses: goanpeca/setup-miniconda@v1
+        with:
+          auto-update-conda: true
+          python-version: ${{ matrix.python }}
+      - name: Conda Config
+        run: |
+          conda config --add channels conda-forge
+          conda config --set channel_priority strict
+          conda config --set always_yes yes
+          conda config --set changeps1 no
+          conda config --set pip_interop_enabled True
+
+      # Installation
```

I'm always tempeted to comment that you shouldn't install into the `base` environment but then remember that the action actually creates and activates a separate environment.
```suggestion
      # Installation into the `test` environment
```